Tests/GeneratedTests/GeneratedTests.swift

```diff
@@ -518,6 +518,12 @@ class InclusiveLanguageRuleGeneratedTests: SwiftLintTestCase {
     }
 }
 
+class IndentationStyleRuleGeneratedTests: SwiftLintTestCase {
```

Comment:
```suggestion
final class IndentationStyleRuleGeneratedTests: SwiftLintTestCase {
```

Reply:
I reverted this change because these are autogenerated tests.